app/src/main/java/com/example/anuj/imdbmovielist/MovieDetail/MovieDetailActivity.java

```diff
@@ -86,120 +67,39 @@ protected void onCreate(Bundle savedInstanceState) {
         recyclerView.setAdapter(videoListAdapter);
         videoListAdapter.setClickListener(this);
 
+        movieDetailPresenter = new MovieDetailPresenter();
+
 
         Results clickedMovie = getIntent().getParcelableExtra("MovieDetailActivity");
```

Comment:
define it **MovieDetailActivity** somewhere in static class so it is accessible and we don't need to retype it.